docs/graphql/core/getting-started/docker-simple.rst

```diff
@@ -63,17 +63,105 @@ Step 3: Open the Hasura console
 
 Head to ``http://localhost:8080/console`` to open the Hasura console.
 
-Step 4: Try Hasura out
+Step 4: Try out Hasura
 ----------------------
 
-Make your :ref:`first graphql query <first_graphql_query>` or set up your :ref:`first event trigger <first_event_trigger>`
+Create a table
+^^^^^^^^^^^^^^
 
-You can also check out our `30-Minute Hasura Basics Course <https://hasura.io/learn/graphql/hasura/introduction/>`__
-and other `GraphQL & Hasura Courses <https://hasura.io/learn/>`__ for a more detailed introduction to Hasura.
+On the Hasura console, navigate to ``Data -> Create table`` and create a sample table called ``profile`` with
+the following columns:
 
-Advanced
---------
+.. code-block:: sql
+
+  profile (
+    id SERIAL PRIMARY KEY, -- serial -> auto-incrementing integer
+    name TEXT
+  )
+
+.. thumbnail:: /img/graphql/core/getting-started/create-profile-table.png
+   :alt: Create a table
+
+Now, insert some sample data into the table using the ``Insert Row`` tab of the ``profile`` table.
+
+Try out a query
+^^^^^^^^^^^^^^^
+
+Head to the ``GraphiQL`` tab in the console and try running the following query:
+
+.. code-block:: graphql
+
+    query {
+      profile {
+        id
+        name
+      }
+    }
+
+You'll see that you get all the inserted data!
+
+.. thumbnail:: /img/graphql/core/getting-started/profile-query.png
+   :alt: Try out a query
+
+Next steps
+----------
+
+Learn course
+^^^^^^^^^^^^
+
+For a full hands-on tour of Hasura, check out our `30-Minute Hasura Basics Course <https://hasura.io/learn/graphql/hasura/introduction/>`__.
+
+Database modelling
+^^^^^^^^^^^^^^^^^^
+
+Check out the :ref:`schema section <schema>` to learn about how to model your database schema, as well as how to extend it.
+
+Querying data
+^^^^^^^^^^^^^
+
+Learn how to use :ref:`GraphQL queries <queries>` to query data from your GraphQL API.
+
+Inserting data
+^^^^^^^^^^^^^^
+
+Learn how to insert data into your GraphQL API using :ref:`GraphQL mutations <mutations>`.
+
+Business logic
+^^^^^^^^^^^^^^
+
+There are several options for the implementation of business logic, depending on your use case.
+
+Actions
+*******
```

Comment:
Irrelevant because we removed most of these titles. But just fyi, it was correct. The titles with `***` were subtitles of others with `^^^` as you can see here:

![image](https://user-images.githubusercontent.com/5722022/98787186-774b7a80-23ff-11eb-8b32-5f6e916240bc.png)